seatunnel-connectors/seatunnel-connectors-spark/seatunnel-connector-spark-feishu/src/main/scala/org/apache/seatunnel/spark/FeishuUtil.scala

```diff
@@ -0,0 +1,149 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.seatunnel.spark
+
+import scala.collection.mutable.ArrayBuffer
+
+import com.alibaba.fastjson.{JSON, JSONArray, JSONObject}
+import org.apache.http.{HttpEntity, HttpHeaders}
+import org.apache.http.client.methods.{CloseableHttpResponse, RequestBuilder}
+import org.apache.http.impl.client.{CloseableHttpClient, HttpClients}
+import org.apache.http.util.EntityUtils
+import org.apache.spark.sql.Row
+import org.apache.spark.sql.types.{DataTypes, StructField, StructType}
+import org.slf4j.{Logger, LoggerFactory}
+
+class FeishuUtil(appId: String, appSecret: String) {
+  val httpClient: CloseableHttpClient = HttpClients.createDefault()
```

Comment:
Rename to FeishuClient is better, and need to add a close method to release the httpClient.

Reply:
Yes, I will change it right now.

Reply:
@kone-net Most look good to me, please rebase the upstream, this will help to fix the UT failed.

Reply:
ok, done.

Reply:
All checks have passed,please approve again.